src/umpire/Umpire.cpp

```diff
@@ -13,8 +13,20 @@
 #include <sstream>
 
 #include "umpire/ResourceManager.hpp"
+#include "umpire/util/MemoryResourceTraits.hpp"
+#include "camp/resource/platform.hpp"
 #include "umpire/config.hpp"
 
+#if defined(UMPIRE_ENABLE_CUDA)
+  #include <cuda_runtime_api.h>
+#endif
+
+//make sure "/opt/rocm-3.6.0/hsa/include/" is 
```

Comment:
Not sure how or if i can edit the blt stuff - is this the best I can do for this?

Reply:
Is this a change that is required in blt?  If so, has an issue been written up there?  I'm not sure these comments belong in the source code.

Reply:
You're right. I wasn't sure how to handle this, but if I didn't add that path, the "hsa/hsa.h" file couldn't be found (needed for hip stuff). I think it's a blt change requirement, but I'm not sure how to make edits to the blt submodule stuff. I agree that the comment doesn't belong there, it was kind of a placeholder for figuring it out later lol

@davidbeckingsale any thoughts?

Reply:
You can now set the CMake variable HIP_RUNTIME_INCLUDE_DIRS - take a look here for an example: https://github.com/LLNL/Umpire/blob/develop/Dockerfile#L69

Reply:
Great, I'll remove this comment then. Thanks!